src/components/SubmitComment.jsx

```diff
@@ -0,0 +1,97 @@
+import { Form, Input, Typography, Button, Space } from 'antd';
+import { useState } from 'react';
+import { postArticleComment } from '../utils/api';
+
+const SubmitComment = ({ article_id, username }) => {
+	const { TextArea } = Input;
+	const { Title } = Typography;
+	const [commentText, setCommentText] = useState('');
+	const [postResponse, setPostResponse] = useState([]);
+
+	const [commentError, setCommentError] = useState('');
+	const [commentSuccess, setCommentSuccess] = useState('');
+
+	const handleInput = (event) => {
+		setCommentText(event.target.value);
+	};
+
+	const handleSubmit = (e) => {
+		setCommentSuccess('Posting your comment...');
+		if (!commentText) {
+			setCommentError('Please enter a comment');
+			setTimeout(() => {
+				setCommentError('');
+			}, 3000);
+		} else {
+			postArticleComment(article_id, username, commentText)
+				.then((response) => {
+					setPostResponse(response);
+					setCommentSuccess('Your comment has been posted');
+					setTimeout(() => {
+						setCommentSuccess('');
+					}, 3000);
```

Comment:
When the comment has been submitted, the commentText does not refresh. The commentText should be an empty string. Consider changing your state 😄